docs/content/docs/flinkDev/ide_setup.md

```diff
@@ -149,6 +149,12 @@ It is recommended to automatically format your code by applying the following se
 8. Under "Formatting Actions", select "Optimize imports" and "Reformat file".
 9. Under "File Path Inclusions", add an entry for `.*\.java` and `.*\.scala` to avoid formatting other file types.
 
+For IntelliJ IDEA version > 2021.2:
+
+6. Go to "Settings" → "Tools" → "Actions on Save".
+7. Under "Formatting Actions", select "Optimize imports" and "Reformat file".
+8. From the "All file types list" next to "Reformat code", select Java and Scala.
```

Comment:
This is fine, but the list above that still mentions the old settings location.

Reply:
I thought about doing it that way but there is an "umbrella" statement that the whole section is applicable to 2020.3. 
https://github.com/afedulov/flink/blob/hotfix-intellij/docs/content/docs/flinkDev/ide_setup.md?plain=1#L65
Do you know if the rest is applicable verbatim in the newer versions? Then we can bump that statement to 2021.2, move the new section up and the existing under the "For older IntelliJ version:" paragraph.

Reply:
AFAICT the `Copyright Profile` doesn't work anymore; doesn't look like it was moved, maybe it was just superceded by file templates (which is what I use).

For scala spotless takes care of license headers, which we could also enabled for java.

The rest should be fine I think.

Reply:
Hmm...I am on 2022.1.3 and the Copyright works fine.

![image](https://user-images.githubusercontent.com/1492164/177995123-02ce78ca-54bf-4ab9-b9e6-fbb3bf9d1044.png)

I am on Ultimate, but [this](https://www.jetbrains.com/help/idea/2022.1/copyright.html) page in the docs does not have the "Ultimate" tag, so it still must be available in the CE.

Reply:
@zentol I adjusted the version as proposed. I checked 2021.2 on both Linux and Mac and the Copyright feature is there.